airbyte-integrations/connectors/source-cin7/manifest.yaml

```diff
@@ -2286,3 +3140,2113 @@ schemas:
+              - "null"
+          TotalBeforeTax:
+            type:
+              - number
+              - "null"
+      SaleOrderDate:
+        type:
+          - string
+          - "null"
+      SalesRepresentative:
+        type:
+          - string
+          - "null"
+      ServiceOnly:
+        type:
+          - boolean
+          - "null"
+      ShippingAddress:
+        type:
+          - object
+          - "null"
+        properties:
+          City:
+            type:
+              - string
+              - "null"
+          Company:
+            type:
+              - string
+              - "null"
+          Contact:
+            type:
+              - string
+              - "null"
+          Country:
+            type:
+              - string
+              - "null"
+          DisplayAddressLine1:
+            type:
+              - string
+              - "null"
+          DisplayAddressLine2:
+            type:
+              - string
+              - "null"
+          ID:
+            type:
+              - string
+              - "null"
+          Line1:
+            type:
+              - string
+              - "null"
+          Line2:
+            type:
+              - string
+              - "null"
+          Postcode:
+            type:
+              - string
+              - "null"
+          ShipToOther:
+            type:
+              - boolean
+              - "null"
+          State:
+            type:
+              - string
+              - "null"
+      ShippingNotes:
+        type:
+          - string
+          - "null"
+      SkipQuote:
+        type:
+          - boolean
+          - "null"
+      SourceChannel:
+        type:
+          - string
+          - "null"
+      Status:
+        type:
+          - string
+          - "null"
+      TaxCalculation:
+        type:
+          - string
+          - "null"
+      TaxRule:
+        type:
+          - string
+          - "null"
+      Terms:
+        type:
+          - string
+          - "null"
+      Transactions:
+        type:
+          - array
+          - "null"
+        items:
+          type:
+            - object
+            - "null"
+          properties:
+            Amount:
+              type:
+                - number
+                - "null"
+            Credit:
+              type:
+                - string
+                - "null"
+            Debit:
+              type:
+                - string
+                - "null"
+            Description:
+              type:
+                - string
+                - "null"
+            EffectiveDate:
+              type:
+                - string
+                - "null"
+            TaskID:
+              type:
+                - string
+                - "null"
+            TransactionID:
+              type:
+                - string
+                - "null"
+      Type:
+        type:
+          - string
+          - "null"
+    required:
+      - ID
+
+api_budget:
```

Comment:
Where is this used?

Reply:
This is a global api rate limiter I had to shove in via yaml editor, it applies to all endpoints.  The api has a 60 call per minute limit, without this it backs off too much and fails.  Honestly, I'd like to put it at the top of the file with a warning comment to put it back if its gone, cause of [issue ](https://github.com/airbytehq/airbyte/issues/56409), but this is what connection builder submitted to github.